tokens/models/tokens/tokens_stablecoins.sql

```diff
@@ -0,0 +1,211 @@
+{{
+    config(
+        schema = 'tokens'
+        , alias = 'stablecoins'
+        , materialized = 'view'
+        , post_hook='{{ expose_spells(blockchains = \'["arbitrum","avalanche_c","base","blast","bnb","celo","ethereum","fantom","gnosis","linea","optimism","polygon","scroll","tron","zkevm","zksync"]\',
+                        spell_type = "sector",
+                        spell_name = "tokens",
+                        contributors = \'["thetroyharris", "gentrexha", "dot2dotseurat", "msilb7", "lgingerich", "Henrystats"]\') }}'
+    )
+}}
+
+SELECT contract_address, symbol, decimals, 'arbitrum' AS blockchain 
```

Comment:
i'm not sure we need to take the hardcoded values approach again. the ideal scenario is to read from `tokens.erc20` and filter on specific tokens. maybe that's a jinja list at top of model of address/blockchain, then filter the main tokens table on the list via a for loop?

Reply:
What is the best approach for me to create a spell that I can use to filter token transfers for contracts that are stablecoins?

The filtered table will then itself be a spell. Trying to do all of this in a query takes too long.

One way or another I have to manually keep a list of contract address that are stablecoins.

Reply:
yes, agree to build an intermediate spell that contains only stablecoins. my ask is that we read from the source `tokens.erc20` and filter that down to stablecoins in this new spell, rather than start from scratch on a hardcoded list. that way we at least keep consistent on symbol/decimals.

you can get fancy with a jinja list variable and loop through that for the filter on the source table, essentially saying:
```
where
    (blockchain = blockchain and address = address)
    OR (blockchain = blockchain and address = address)
    ...
```

Reply:
that would be preferred approach. if the variable isn't working, you can also do the `union all` approach and filter on a blockchain/addresses per union'd query

Reply:
i would expect the files modified in this PR to be much fewer to get this built